packages/pds-ember/app/styles/pds/components/link/default/__config.scss

```diff
@@ -0,0 +1,51 @@
+$_module: "PDS.Components.Link.Default";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "sass:map";
+@use "../__config" as Link;
```

Comment:
FYI: 

Config files are intentionally (re)named with a double-underscore prefix, so that Sass does not map `config` with `__config`.  Sass will still associate `_config` (one underscore) with `__config.scss` (a.k.a. "dunder" config), but I'm using both underscores to avoid confusion with Sass partials.  This forces the use of at least one underscore in the path to communicate the use of a private module.

The dunder config will expose private members for styling, while an optional `utils.scss` can forward or alias private members for public use.  Exposed members in `utils.scss` need to remain as stable and backward-compatible as possible, while `__config.scss` member may change at any time.